net-p2p/mldonkey/mldonkey-3.1.7.ebuild

```diff
@@ -0,0 +1,187 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit flag-o-matic eutils autotools toolchain-funcs
+
+IUSE="bittorrent doc fasttrack gd gnutella gtk guionly magic +ocamlopt upnp"
+
+DESCRIPTION="Multi-network P2P application written in Ocaml, with Gtk, web & telnet interface"
+HOMEPAGE="http://mldonkey.sourceforge.net/"
```

Comment:
Please also add the Github page here.

Reply:
Well, actually, this is something I didn't know what to do with it.
The SF page is still the 'official' one and it's still linked from the GH code, but the GH code is way ahead the SF project which is unmaintained. Even the developer is different. Is it like the SF project was abandoned and somebody took it and continued the work in GH and the SF site was left abandoned (still linking to older releases) but it's still the place with most of the documentation.
I will take your advice and add both sites to `HOMEPAGE`.